docs(wandb): explain how to use W&B integration (#5607)
huggingface/transformers: examples/README.md

```

Feedback and more use cases and benchmarks involving TPUs are welcome, please share with the community.

> ## Logging experiments with Weights & Biases

Reviewer: ```suggestion
## Logging & Experiment tracking
```

> You can easily log and monitor your runs code with [Weights & Biases](https://docs.wandb.com/library/integrations/huggingface).

Reviewer: ```suggestion
You can easily log and monitor your runs code. [TensorBoard](https://www.tensorflow.org/tensorboard) and [Weights & Biases](https://docs.wandb.com/library/integrations/huggingface) are currently supported.
```

> Install wandb with:

Reviewer: ```suggestion
To use Weights & Biases, install the wandb package with:
```
